packages/flutter_tools/lib/src/resident_runner.dart

```diff
@@ -251,14 +251,15 @@ class FlutterDevice {
     return _views;
   }
 
-  Future<void> exitApps() async {
+  Future<void> exitApps({
+    @visibleForTesting Duration timeoutDelay = const Duration(seconds: 10),
```

Comment:
Should we also set this to zero for https://github.com/flutter/flutter/blob/bd897dfafc65042e175e0f860d376a562e31ba31/packages/flutter_tools/test/general.shard/resident_runner_test.dart#L812?

Is there anything we can do to ensure future tests also override this?

Reply:
I don't think there is a requiredButOnlyForTesting ... that would be neat. In this case, we don't hit that code path with the timeout, but if we did the test would fail since it would receive an unexpected vm service request for ext.flutter.exitApp

Reply:
ahh, good point.